homeassistant/components/group/entity.py

```diff
@@ -368,9 +369,23 @@ def async_update_group_state(self) -> None:
         self._state = None
         self._async_update_group_state()
 
+    async def async_domain_registered(self, domain: str) -> None:
+        """Refresh after a new entity platform has been registered."""
+        _LOGGER.debug(
+            "Domain %s was registered, refreshing state %s",
+            domain,
+            self.entity_id,
+        )
+        await self.async_update_tracked_entity_ids(set(self.tracking))
```

Comment:
This looks like it doesn't need to be a coro since nothing is awaited here.

That will also prevent the dispatcher from creating lots of tasks